core/src/main/java/org/stellar/anchor/event/models/Amount.java

```diff
@@ -0,0 +1,15 @@
+package org.stellar.anchor.event.models;
```

Comment:
These classes should belong to the `platform-api`. We may merge this and move the classes in the next PR.

Reply:
will refactor in a separate PR